enterprise/packages/billing/package.json

```diff
@@ -47,5 +48,8 @@
     "@nestjs/swagger": "^7.1.8",
     "@nestjs/throttler": "^5.0.1",
     "@novu/dal": "workspace:*"
+  },
+  "optionalDependencies": {
+    "@novu/ee-auth": "workspace:*"
```

Comment:
This will be changed to direct dependency when we remove `CLERK_ENABLED` flag, since EE === CLERK_ENABLED